### ISSUE TITLE
Does not work for me on mac

### ISSUE THREAD

**Issue by EdFenner** [yekingyan/obsidian-open-in-other-editor#7]
I put in the path. Tried different ways.  Does nothing.

**Comment by mariomui** [comment]
Are you on windows?

**Comment by mariomui** [comment]
Can I see the path that you put in?

**Comment by EdFenner** [comment]
The last path was: /Applications/Visual Studio Code.app

**Comment by mariomui** [comment]
go to terminal and type `which code` 
you should see something like this.  If note
/usr/local/bin/code 

If you dont see anything, you haven't linked the code shell command in your path so...
https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line

**Comment by EdFenner** [comment]
That is exactly what I see:
/usr/local/bin/code

**Comment by mariomui** [comment]
that's the path that you should put in.

**Comment by EdFenner** [comment]
Ah! Well that changes everything.  In most cases I have had to give the path to the app that it was trying to communicate with, not the bin.  I would suggest making that clear.

**Comment by mariomui** [comment]
yeah, i'll do that. let me finish refactor this open folder in vscode feature first.

**Comment by EdFenner** [comment]
This is really helpful, just what I was looking for - thanks!!
